plugins/SolarSystemEditor/src/gui/solarSystemManagerWindow.ui

```diff
@@ -132,7 +132,7 @@
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>0</number>

```

Comment:
First window should open by default

Reply:
The main dialog is not the MPC import dialog, but the actual editor window - that's the name of the plugin. In that editor, a button allows to open the MPC import dialog, a secondary dialog.

Actually, the second tab ("Solar System") should be swapped with the first tab, because that is the **core tab** of the SSE. It also draws the attention to the MPC import button.

The "configuration file" tab is a less important and less used tab that contains (dangerous) actions that are seldom used. That's why I propose to make the "Solar System" tab open by default.

Also, SSE should receive the same treatment as other plugins, i.e. a toolbar button that opens a main window.

Reply:
This is not reasons for me. @gzotti ?

Reply:
OK. if you want, we can discuss this. (about a single change that nobody else will complain about, I'm rather sure about that. )

you ask me for my motivation, I've written it down in detail, and it sounds quite reasonable to me.

remember that the pages can be swapped in Qt Editor. I'm more than happy to do that, I already tested it, but I just need confirmation for that.

now I want to know what your motivation is.

"_it has been like that before_" or "_nobody complained about it_" are no good reasons. that's not the way software evolves. maybe I oversaw something?

Reply:
I don't want discuss anything and I don't like breakage behaviour of the plugin. Just check GUI and behaviour of all other plugins and say us where is you wrong in your "reasons".